blazor/datepicker/style-appearance.md

```diff
@@ -35,6 +35,17 @@ Use the following CSS to customize the DatePicker icon element
 }
 ```
 
+## Adding backgroud color to DatePicker container element
+
+Use the following CSS selector, can add color to DatePicker container element.
```

Comment:
Use the following CSS selector to add color to the DatePicker container element.